interop/tapir/src/main/scala/caliban/interop/tapir/package.scala

```diff
@@ -162,17 +162,19 @@ package object tapir {
     override protected val additionalDirectives: List[__Directive] = Nil
   }
 
-  private def extractPath[I](input: EndpointInput[I]): String =
-    input
-      .asVectorOfBasicInputs(includeAuth = false)
-      .collect {
-        case EndpointInput.FixedPath(s, _, _) => s
+  private def extractPath[I](endpointName: Option[String], input: EndpointInput[I]): String =
+    endpointName.getOrElse(
+      input
+        .asVectorOfBasicInputs(includeAuth = false)
+        .collect {
+          case EndpointInput.FixedPath(s, _, _) => s
```

Comment:
I'm wondering if we should catch `PathCapture` here and use the name for the field. With your example I imagine there would be a `PathCapture` with name `orderId`, so maybe we could generate a field `ordersByOrderId` or `ordersWithOrderId`?